Source/NSubstitute/SubstituteExtensions.cs

```diff
@@ -222,7 +294,25 @@ internal static ConfiguredCall Returns<T>(MatchArgs matchArgs, Func<CallInfo, T>
         public static IEnumerable<ICall> ReceivedCalls<T>(this T substitute) where T : class
         {
             return GetRouterForSubstitute(substitute).ReceivedCalls();
-        }       
+        }
+
+#if NET4 || NET45
```

Comment:
Internal `TaskHelper` isn't that bad imo :smile: I prefer to keep classes concise and think managing condition compilation over whole file/class (or small class) is easier than somewhere at the end of big file.